appeals/web/src/server/appeals/appeal-details/costs/costs.mapper.js

```diff
@@ -40,53 +39,74 @@ export function addDocumentTypePage(appealDetails, documentTypes) {
+ * @returns {PageContent}
  */
-export function documentUploadPage(
-	appealDetails,
-	costsApplicant,
-	folderId,
-	folderPath,
-	documentType,
-	errors
-) {
-	const pathComponents = folderPath.split('/');
-	const documentStage = pathComponents[0];
-
-	let costsApplicantLabel = '';
-
-	switch (costsApplicant) {
-		case 'appellant':
-			costsApplicantLabel = 'Appellant ';
-			break;
-		case 'lpa':
-			costsApplicantLabel = 'LPA ';
-			break;
-	}
+export function decisionCheckAndConfirmPage(appealDetails, decisionDocumentFolder) {
+	const shortAppealReference = appealShortReference(appealDetails.appealReference);
 
-	const pageTitleText = `Upload ${costsApplicantLabel}costs document`;
+	/** @type {PageComponent} */
+	const summaryListComponent = {
+		type: 'summary-list',
+		parameters: {
+			rows: [
+				{
+					key: {
+						text: 'Costs decision'
+					},
+					value: {
+						html: `<ul class="govuk-list"><li>${(decisionDocumentFolder.documents || [])
+							.map(
+								(document) =>
+									`<a class="govuk-link" href="/documents/${appealDetails.appealId}/download/${document.id}/preview/">${document.name}</a>`
+							)
+							.join('</li><li>')}</li></ul>`
+					},
+					actions: {
+						items: [
+							{
+								text: 'Change',
+								href: `/appeals-service/appeal-details/${appealDetails.appealId}/costs/decision/upload-documents/${decisionDocumentFolder.id}`
+							}
+						]
+					}
+				}
+			]
+		}
+	};
 
-	return {
-		backButtonUrl: `/appeals-service/appeal-details/${appealDetails.appealId}/costs/${costsApplicant}/add`,
-		appealId: `${appealDetails.appealId}`,
-		folderId: folderId,
-		useBlobEmulator: config.useBlobEmulator,
-		blobStorageHost:
-			config.useBlobEmulator === true ? config.blobEmulatorSasUrl : config.blobStorageUrl,
-		blobStorageContainer: config.blobStorageDefaultContainer,
-		multiple: false,
-		documentStage: documentStage,
-		pageTitle: pageTitleText,
-		appealShortReference: appealShortReference(appealDetails.appealReference),
-		pageHeadingText: pageTitleText,
-		documentType: documentType,
-		nextPageUrl: `/appeals-service/appeal-details/${appealDetails.appealId}/costs/${costsApplicant}/add-document-details/`,
-		errors
+	/** @type {PageContent} */
+	const pageContent = {
+		title: `Check your answers - ${shortAppealReference}`,
+		backLinkUrl: `/appeals-service/appeal-details/${appealDetails.appealId}/costs/decision/upload-documents/${decisionDocumentFolder.id}`,
+		heading: 'Check your answers',
+		pageComponents: [
+			...(decisionDocumentFolder.documents && decisionDocumentFolder.documents.length > 0
+				? [summaryListComponent]
+				: []),
+			{
+				type: 'warning-text',
+				parameters: {
+					text: 'You must email the relevant parties to inform them of the decision.'
+				}
+			},
+			{
+				type: 'checkboxes',
+				parameters: {
+					name: 'confirm',
+					items: [
+						{
+							text: 'I will email the relevant parties',
+							value: 'yes',
+							checked: false
```

Comment:
I don't think you need to specify checked: false? I think it defaults to that.

Reply:
Yep it does, good catch!